rust/connlib/tunnel/src/client.rs

```diff
@@ -737,10 +734,12 @@ impl ClientState {
             return Err(Error::ControlProtocolError);
         };
 
+        tracing::info!(domain = %domain_response.domain, addresses = ?domain_response.address, "Received DNS response");
```

Comment:
This _might_ be better as `debug`. DNS queries can be considered by some to be sensitive info, so it would be better to keep them off the disk by default.

```suggestion
        tracing::debug!(domain = %domain_response.domain, addresses = ?domain_response.address, "Received DNS response");
```

Reply:
These are just for DNS resources though but happy to move it to debug!

Reply:
Actually, I think these are also regular DNS requests so will move to debug.